Goal: Better German translation (attempt)
Fixes: #176408

Status wrt upstream: Not applied and not proposed

Index: geneweb-5.00/hd/lang/lexicon.txt
===================================================================
--- geneweb-5.00.orig/hd/lang/lexicon.txt	2007-01-19 02:53:16.000000000 +0100
+++ geneweb-5.00/hd/lang/lexicon.txt	2007-02-28 21:05:11.589249483 +0100
@@ -1489,7 +1489,7 @@
 ca: després de
 cs: po
 da: efter
-de: nach
+de: nachher
 en: after
 eo: post
 es: después de
@@ -2108,7 +2108,7 @@
 ca: abans de
 cs: pøed
 da: før
-de: vor
+de: vorher
 en: before
 eo: antaù
 es: antes
